src/richie/apps/core/templates/richie/base.html

```diff
@@ -66,48 +66,61 @@
+        {# Access key directive allows users to use keyboard shortcuts #}
+        <a href="#site-content" class="sr-only sr-only-focus" accesskey="c">{% trans "Skip to main content" %}</a>
+        
             {% block body_header %}
-            <nav class="topbar {% block topbar_classes %}{% endblock topbar_classes %}" id="main-menu">
-                <div class="topbar__container">
-                    <div class="topbar__brand">
-                        <button class="topbar__hamburger"
-                                data-target="main-menu" aria-label="menu" aria-expanded="false">&#8801;</button>
-                        <a href="/">
-                            <img src="{% static "richie/images/logo.png" %}" class="topbar__logo" alt="{{ SITE.name }}">
-                        </a>
-                    </div>
-                    <div class="topbar__menu">
-                        <ul class="topbar__list">
-                            {% show_menu 0 100 0 0 "menu/header_menu.html" %}
-                        </ul>
-                    </div>
-                    <div class="topbar__menu topbar__menu--aside">
-                        {% block topbar_searchbar %}
-                        <div class="topbar__search richie-react richie-react--root-search-suggest-field"
-                            data-props='{"courseSearchPageUrl": "{% page_url 'courses' %}"}'></div>
-                        {% endblock topbar_searchbar %}
-
-                        <ul class="topbar__list topbar__list--controls">
-                            {% block userlogin %}
-                                {% if AUTHENTICATION %}
-                                    <li class="topbar__item topbar__item--login richie-react richie-react--user-login" data-props='{"profileUrls": {{ AUTHENTICATION.profile_urls }}}'></li>
-                                {% endif %}
-                            {% endblock userlogin %}
-                            {% block topbar_contact %}
-                            <li class="topbar__item topbar__item--cta">
-                                <a href="#">{% trans "Contact us" %}</a>
-                            </li>
-                            {% endblock topbar_contact %}
-                        </ul>
-
-                        {% language_chooser "menu/language_menu.html" %}
+
+            {# Wrap header content in a header tag to improve landmarks around the site #}
+
+            <header id="site-header">
+                <nav class="topbar {% block topbar_classes %}{% endblock topbar_classes %}" id="main-menu">
+                    <div class="topbar__container">
+                        <div class="topbar__brand">
+                            <button class="topbar__hamburger"
+                                    data-target="main-menu" aria-label="menu" aria-expanded="false">&#8801;</button>
+                            <a href="/" title="{% trans "Go to homepage" %}" rel="home" accesskey="h">
```

Comment:
I think the correct way to do this is to just add "Home" to the alt text for the logo, not add a title attribute for the link

Reply:
From my experience, accessibility wise, you shouldn't have elements without context. You have a img element wrapped in an anchor tag, but the screen reader will read and list the link and not the element inside it. NVDA and JAWS will internally create a link index for all links with the link itself and the text inside it (context). We can either do this or opt for a `<span class="sr-only">SITE TITLE</span>` inside the link, to give it context. I've been using https://accessmonitor.acessibilidade.gov.pt/ to do validation, along with Deque AXE and WAVE.

Reply:
I don't get what "context" is here.

This is actually something we used to do wrong in Richie. We used to put titles on all links and when we did an accessibility audit, the auditor warned us that is was not good practice, as link content is sufficient and titles are just additional noise (except for some cases such as file downloads).

For a link that just contains an image, alt text on the image is sufficient as it is the content of the link and the link's name in the accessibility tree.

Reply:
You're absolutely right @mbenadda. An image with an alt tag will do the job. For this specific link I just like to add better context. At the moment, the "homepage" link says the name of the site as defined in the configurations. I like to have a bit more context in this homepage link, because of its importance. Having a direct access key and a title that describes what it does makes sense to me, due to my experience in this are. We can refactor it and leave the title out or leave it as it was previously. Sorry if I explained myself wrong and thank you for your feedback.

Reply:
@mbenadda we discussed this irl and it's true that, for a screen reader, the most important thing about this block is that it links back to the homepage. The "alt" as it is does not bring any information...

Maybe a solution could be to keep the "title" with the call to action "Go to home page" and set the image alt to an empty string?

Other solutions seem less good. Removing the title and setting the image "alt" to:
- "Go to home page" does not make sense because the image is a logo and has nothing to do with the action assigned to this image by the link...
- "{{ SITE.name }} logo" does not bring any value to the screen reader user

Reply:
I think setting the alt to an empty string and using the title might be a RGAA non-conformity.

What about changing the alt of the image to "{{ SITE.name }} home"?

Reply:
Like I said previously, Your solution is correctly. The documentation confirms it (https://www.w3.org/WAI/WCAG21/Techniques/failures/F89.html). The only note a leave is that this is a special case. The problem with title is that, most times, it repeats the alt text, so blind users ear the same text twice. In this case, this is my proposed solution, and the one I'll implement on our end.

```html
<!-- this gives context to my action, navigation to the homepage. you have aditional access via keyboard shortcut and the rel microformat as described in http://microformats.org/wiki/existing-rel-values#HTML5_link_type_extensions -->
<a href="/" title="{% trans "Go to homepage" %}" rel="home" accesskey="h">
    <!-- this can be an image or an inline svg (in our case). it provides a bit more flexibility, because i'm not bound to the content inside the link-->
    <img src="<image source>" alt="Institution"> <!-- the alt here is still the image description, as a complement to the previous -->
</a>
```
As I said, I can regress this to a previous version or add a `home` at the end of `alt` and then make the changes on our theme. Will be waiting for you feedback. Thank you.

Reply:
Ok, we can implement this in Richie 👍

No need to theme this on your end.